build-hazelcast-homebrew-package.sh

```diff
@@ -58,7 +58,7 @@ function generateFormula {
   all_hz_versions=({hazelcast.rb,hazelcast?[0-9]*\.rb,hazelcast-enterprise*\.rb})
   for version in "${all_hz_versions[@]}"
   do
-    if [[ "$version" != "$file" ]] && [[ ! "$version" =~ .*(beta|devel).* ]] ; then
+    if [[ "$version" != "$file" ]] && [[ ! "$version" =~ .*(beta).* ]] ; then
```

Comment:
```suggestion
    if [[ "$version" != "$file" ]] && [[ ! "$version" =~ .*beta.* ]] ; then
```

Nit - probably don't need the brackets.

Reply:
Good catch. I have also removed additional unwanted brackets

Fixed [68f9047]( https://github.com/hazelcast/hazelcast-packaging/pull/224/commits/68f90478f006216fb220b22cc2b4f892d4db9111)